### ISSUE TITLE
No module named 'timm.models.efficientnet_builder'

### ISSUE THREAD

**Issue by Dhatri2012** [zetyquickly/DensePoseFnL#20]
Hi while running test command getting below error
Command executed 
python train_net.py --config-file configs/mobile_parsing_rcnn_b_s3x.yaml --eval-only MODEL.WEIGHTS mobile_parsing_rcnn_b_s3x.pth
python train_net.py --config-file configs/mobile_parsing_rcnn_b_s3x.yaml --eval-only MODEL.WEIGHTS mobile_parsing_rcnn_b_cse_s3x.pth
python train_net.py --config-file configs/mobile_parsing_rcnn_b_s3x.yaml --eval-only MODEL.WEIGHTS mobile_parsing_rcnn_b_wc2m_s3x.pth

 python train_net.py --config-file configs/mobile_parsing_rcnn_b_s3x.yaml --eval-only MODEL.WEIGHTS mobile_parsing_rcnn_b_wc2m_s3x.pth
Traceback (most recent call last):
  File "/teamspace/studios/this_studio/VirtualTryOn/paintByEx/DensePose2/DensePoseFnL/train_net.py", line 24, in <module>
    import backbone
  File "/teamspace/studios/this_studio/VirtualTryOn/paintByEx/DensePose2/DensePoseFnL/backbone.py", line 9, in <module>
    from timm.models.efficientnet_builder import BN_MOMENTUM_TF_DEFAULT, BN_EPS_TF_DEFAULT
ModuleNotFoundError: No module named 'timm.models.efficientnet_builder'

**Comment by zetyquickly** [comment]
timm==0.4.12 ?

**Comment by jeevitesh** [comment]
Above issue got resolved after installing 0.4.12
now facing below issue please help

File "/paintByEx/DensePose2/DensePoseFnL/train_net.py", line 77, in <module>
    launch(
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.10/site-packages/detectron2/engine/launch.py", line 84, in launch
    main_func(*args)
  File "/paintByEx/DensePose2/DensePoseFnL/train_net.py", line 54, in main
    res = Trainer.test(cfg, model)
  File "/paintByEx/DensePose2/detectron2/projects/DensePose/densepose/engine/trainer.py", line 113, in test
    data_loader = cls.build_test_loader(cfg, dataset_name)
  File "/paintByEx/DensePose2/detectron2/projects/DensePose/densepose/engine/trainer.py", line 217, in build_test_loader
    return build_detection_test_loader(cfg, dataset_name, mapper=DatasetMapper(cfg, False))
  File "/paintByEx/DensePose2/detectron2/projects/DensePose/densepose/data/build.py", line 480, in build_detection_test_loader
    dataset_dicts = combine_detection_dataset_dicts(
  File "/paintByEx/DensePose2/detectron2/projects/DensePose/densepose/data/build.py", line 390, in combine_detection_dataset_dicts
    dataset_name_to_dicts[dataset_name] = DatasetCatalog.get(dataset_name)
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.10/site-packages/detectron2/data/catalog.py", line 58, in get
    return f()
  File "/paintByEx/DensePose2/detectron2/projects/DensePose/densepose/data/datasets/coco.py", line 405, in load_annotations
    return load_coco_json(
  File "/paintByEx/DensePose2/detectron2/projects/DensePose/densepose/data/datasets/coco.py", line 367, in load_coco_json
    coco_api = _load_coco_annotations(PathManager.get_local_path(annotations_json_file))
  File "/paintByEx/DensePose2/detectron2/projects/DensePose/densepose/data/datasets/coco.py", line 170, in _load_coco_annotations
    coco_api = COCO(json_file)
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.10/site-packages/pycocotools/coco.py", line 81, in __init__
    with open(annotation_file, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'datasets/coco/annotations/densepose_minival2014.json'

**Comment by zetyquickly** [comment]
make sure that the densepose dataset is placed where it's expected

```
FileNotFoundError: [Errno 2] No such file or directory: 'datasets/coco/annotations/densepose_minival2014.json'
```
